msg/ExternalCollisionProtectionNameList.msg

```diff
@@ -0,0 +1,9 @@
+# A message for setExternalCollisionProtectionEnabled in External-Collision avoidance API
+
+uint8 number
```

Comment:
I would prefer either the usual "data" used in ROS, or a name related to the information you transmit (for instance "protected_parts"). But "data" would really be the best here I think. 
I find "number" misleading (one might think that it is the number of protected parts or whatever). 

Besides, maybe the name of the file should be changed ? It is not a "Name List" anymore :)